projects/angular/src/lib/maskito.directive.ts

```diff
@@ -30,16 +30,16 @@ export class MaskitoDirective implements OnDestroy, OnChanges {
 
     @Input()
     maskitoElement: MaskitoElementPredicate = e =>
-        e.querySelector('input,textarea') ||
+        e.querySelector<HTMLInputElement | HTMLTextAreaElement>('input,textarea') ||
```

Comment:
This is unnecessary, type is inferred from the `MaskitoElementPredicate` type of the arrow function.